src/components/lab/canvas/formulas/CarbonDioxideFormula.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import MoleculeFormula from './MoleculeFormula';
+
+const CarbonDioxideFormula = () => {
+  const { t } = useTranslation();
+  return (
+    <MoleculeFormula
+      formula={
+        // eslint-disable-next-line react/jsx-wrap-multilines
+        <>
+          {t('(CO')}
```

Comment:
Maybe for something simpler you can check greenhouse-effect where I get advantage of `t` to format the `sub`. It also avoids to have 3 lines for one formula.

Reply:
Additionally, you can factor the inner component above which will not trigger eslint. (but with my comment above you won't even have to do it)